unison-share-api/src/Unison/Server/Backend.hs

```diff
@@ -1096,9 +1101,9 @@ scopedNamesForBranchHash codebase mbh path = do
     Nothing
       | shouldUseNamesIndex -> indexNames
       | otherwise -> do
-          rootBranch <- lift $ Codebase.getRootBranch codebase
-          let (parseNames, _prettyNames, localNames) = namesForBranch rootBranch (AllNames path)
-          pure (parseNames, localNames)
+        rootBranch <- lift $ Codebase.getRootBranch codebase
```

Comment:
If possible it'd be great if you could upgrade your ormolu to 4.0.0.0 to avoid formatter battles; it's what all of the UCM team are using at the moment 😄